src/tools/plotter/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(iDynTreePlotter)
+
+find_package(Qt5 COMPONENTS Widgets Charts)
```

Comment:
In iDynTree we are currently using strict CMake option to enable/disable dependencies (see https://github.com/robotology/idyntree/blob/add-sole-gui/cmake/iDynTreeOptions.cmake#L58 ), similarly to YARP/ICUB so you get a firm error if you requested a functionality for which you don't have a dependency. 
While this may be undesirable, as you need to (manually|in the superbuild) specificy the dependencies that you have rather then  automatically discovery them, but if we change I would change all the dependencies to work in that way, rather then just switch a subproject. 

In particular, in this case this would break the build for anyone that has Qt5 installed but do not have YARP enabled.